trajectory_msgs/JointTrajectory joint_trajectory
string trajectory_name
---
bool success
